setup.py

```diff
@@ -56,6 +55,11 @@
     ),
     install_requires=REQUIREMENTS,
     include_package_data=True,
-    python_requires=">=3.6",
+    python_requires=">=3.7",
```

Comment:
I assume this in line with other Qiskit packages? Else I don't see why 3.6 shouldn't be sufficient.

Reply:
This is in line with terra: https://github.com/Qiskit/qiskit-terra/blob/main/setup.py